docs/overview.yaml

```diff
@@ -139,6 +139,10 @@ General algebra:
   Transcendental numbers:
     Liouville's theorem on existence of transcendental numbers: 'transcendental_liouvilleNumber'
 
+  Algebraic Number Theory:
+    Finiteness of the class number: 'NumberField.RingOfIntegers.instFintypeClassGroupSubtypeMemSubalgebraIntToCommSemiringInstCommRingIntToSemiringToCommSemiringToCommRingToEuclideanDomainAlgebraIntToRingToDivisionRingInstMembershipInstSetLikeSubalgebraRingOfIntegersToCommRingIsDomainToIsDomainIsDedekindDomainIsDomainTo_principal_ideal_domain'
```

Comment:
Hmm, I would much prefer that we rename this celebrated result to something a bit more readable. Would you mind doing that in this same PR?